prep-javascript-dom-events/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Events</title>
+</head>
+<body>
+  <button class="hello-world-btn">Hello World!</button>
+  <script>
+    document.addEventListener('DOMContentLoaded', function () {
+    });
+    console.log('content loaded!');
```

Comment:
We want this `console.log()` to only run when the page finishes loading, so we'll need to move it into the function's code block.

Reply:
![image](https://user-images.githubusercontent.com/117887912/204103919-b11cb85f-b52a-49fb-9ba2-2e2b7bd977bc.png)